pkg/volume/util/fsquota/quota_linux.go

```diff
@@ -220,6 +219,13 @@ func getQuotaOnDir(m mount.Interface, path string) (common.QuotaID, error) {
 	return getApplier(path).GetQuotaOnDir(path)
 }
 
+func getQuotaIDFromDirPath(path string) (common.QuotaID, bool) {
+	if val, ok := dirQuotaMap[path]; ok {
```

Comment:
`dirQuotaMap` won't be populated at kubelet start -- it only gets populated by `assignQuota`.  You need to use `GetQuotaOnDir` here.

Reply:
Done